types/node/ts4.8/zlib.d.ts

```diff
@@ -110,9 +110,15 @@ declare module "zlib" {
         memLevel?: number | undefined; // compression only
         strategy?: number | undefined; // compression only
         dictionary?: NodeJS.ArrayBufferView | ArrayBuffer | undefined; // deflate/inflate only, empty dictionary by default
-        info?: boolean | undefined;
         maxOutputLength?: number | undefined;
     }
+    interface ZlibOptionsWithoutInfo extends ZlibOptionsBase {
+        info?: false | undefined;
+    }
+    interface ZlibOptionsWithInfo extends ZlibOptionsBase {
+        info: true;
+    }
+    type ZlibOptions = ZlibOptionsWithoutInfo | ZlibOptionsWithInfo;
```

Comment:
This change in declaration is a breaking change; is there a downside to leaving this behind as the base instead?

Reply:
I understand it is a breaking change, just now made some changes to so that the `ZlibOptions` interface itself is now unaffected, unfortunately this PR does remain a breaking change but that is just because the initial definition just didn't include the info object that the Node.JS documentation describes which it should have.